docs/addons.md

```diff
@@ -64,7 +64,9 @@ configuration.
 
 **knative**: Adds the [Knative][knative-docs] middleware to your cluster.
 
-**[linkerd](/docs/addon-linkerd)**: Deploys the [linkerd][linkerd-docs] service mesh.
+[**Kubeflow**](addon-kubeflow): The easiest way to access [Kubeflow][], the machine learning toolkit for Kubernetes.<span class="p-label--new">New in 1.16.2 </span>
```

Comment:
Kubeflow will not be available on 1.16 track, so we should set the label to "New in 1.17.0, available also in latest stable"